src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -366,7 +367,7 @@ private fn_header ::= CONST? UNSAFE? [ EXTERN abi? ] FN IDENTIFIER
 
 fn_params ::= LPAREN !COMMA params COMMA? RPAREN { pin(".*") = 1 }
 private params ::= [ param (COMMA param)* ]
-param ::= pat COLON type_sum
+param ::= pat COLON type
 
 ret_type  ::= ARROW (type | EXCL) { pin = 1 }
```

Comment:
This guy has pretty unfortunate name. Maybe `ret_type_spec`? Rust Reference calls this *output type* which is even worse in out case.